simapp/simd/cmd/root.go

```diff
@@ -157,6 +157,16 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 	cfg := sdk.GetConfig()
 	cfg.Seal()
 
+	creator := func(logger log.Logger, db dbm.DB, options servertypes.AppOptions) (servertypes.RootMultiStoreProvider, error) {
+		pruningOptions, err := server.GetPruningOptionsFromFlags(options)
+		if err != nil {
+			return nil, err
+		}
+
+		app := simapp.NewSimApp(logger, db, nil, true, encodingConfig, nil, baseapp.SetPruning(pruningOptions))
```

Comment:
Can reuse the exiting `newApp` function? that'll need to use `servertypes.Application` interface, but since all the cosmos-sdk scaffolding chains has the appCreator, I think it's fine, and keep the code simpler.

Reply:
I define new interface because`servertypes.Application` interface does not include function like `CommitMultiStore()`.
It seems like a breaking change to add `CommitMultiStore()` function to the existing interface.
I can add this function if breaking change is ok

Reply:
It's ok to break api in the main branch

Reply:
add `CommitMultiStore()` to `AppCreator`